artifactory_test.go

```diff
@@ -337,10 +338,9 @@ func testArtifactoryDownload(fileSize int, t *testing.T) {
 	localFileDetails, err := fileutils.GetFileDetails(randFile.Name(), true)
 	assert.NoError(t, err)
 
-	artifactoryCli.Exec("u", randFile.Name(), tests.RtRepo1+"/testdata/", "--flat=true")
-	randFile.File.Close()
```

Comment:
I think this line was mistakenly removed.

Reply:
i replaced `artifactoryCli.Exec("u", ... ` with `runRt(t, "u", ...`
and the `randFile.File.Close()` is removed because randfile is never opened so after i added the assertions, the tests failed on all those randfile.close lines.